.github/workflows/pr-title.yml

```
@@ -0,0 +1,14 @@
+on:
+  pull_request:
+    types: [opened, synchronize, reopened, edited]
+
+jobs:
+  validate-pr-title:
+    runs-on: ubuntu-latest
+    steps:
+      - name: PR Conventional Commit Validation
+        uses: ytanikin/PRConventionalCommits@1.1.0
+        with:
+          task_types: '["feat","fix","docs","test","ci","refactor","perf","chore","revert", "build"]'
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)